src/spikeinterface/extractors/tests/test_nwbextractors.py

```diff
@@ -190,20 +194,22 @@ def test_sorting_extraction_of_ragged_arrays(tmp_path):
 
     # Add the spikes
     nwbfile.add_unit_column(name="unit_name", description="the name of the unit")
+    nwbfile.add_unit_column(name="a_property", description="a_cool_property")
+
     spike_times1 = np.array([0.0, 1.0, 2.0])
-    nwbfile.add_unit(spike_times=spike_times1, unit_name="a")
+    nwbfile.add_unit(spike_times=spike_times1, unit_name="a", a_property="a_property_value")
     spike_times2 = np.array([0.0, 1.0, 2.0, 3.0])
-    nwbfile.add_unit(spike_times=spike_times2, unit_name="b")
+    nwbfile.add_unit(spike_times=spike_times2, unit_name="b", a_property="b_property_value")
 
-    ragged_array_bad = [[1, 2, 3], [1, 2, 3, 5]]
+    ragged_array_bad = [[1, 2, 3, 8, 10], [1, 2, 3, 5]]
```

Comment:
Out of curiosity why is this `ragged_array_bad` this is actually a ragged array whereas `ragged_array_good` is technically not ragged right? I understand 'bad' because it is ragged, but then wouldn't it be a little better to call the "bad" array

```
ragged_array = [[1,2,3,8,10],[1,2,3,5]] # ragged array doesn't work with nwb
...
standard_array = [[1,2,3],[4,5,6]] # standard array works fine with nwb.
```

This doesn't affect the tests, but wouldn't it be better naming to not call it `ragged_array_bad` if it is is a good ragged array, which is bad for the structure?

Reply:
The ragged array bad is bad because it does not work with spikeinterface. Not the other way around. Both ragged arrays good and bad work fine in nwb.

We could have better naming though.

Reply:
Aw. Cool thanks. My main point was on the naming, but since I don't use nwb too much (yet), I wasn't sure if the problem (with being ragged) was nwb or si. Thanks for the clarification.

Reply:
I mean, you are correct, the namig is ambigious in that way. It could be better. I will think on something.

Reply:
I added more meaningful names, @zm711.

Reply:
Thanks @h-mayorquin!